_episodes/02-setup.md

```diff
@@ -98,6 +98,8 @@ same commands to choose another editor or update your email address.
 > $ git config --global --unset https.proxy
 > ~~~
 > {: .bash}
+> In addition git supports the SSH protocol, which needs no proxy configuration, and
```

Comment:
I'd prefer not to include this change. I think it's useful information for a more advanced crowd, but it may be confussing for a novice. The only reason the proxy callout is here is because it may be needed in some workshops with a weird network configuration.